### PR TITLE
WIP: PrestoSyntax

### PR DESCRIPTION
**Pull request by commercial-hippie**
Busy testing this myself, still a work in progress.. Will update as I go.

### CODE REVIEW

**Comment by pyongjoo** [comment]
Since we have received an email confirmation, @dongyoungy will merge this into the master.

File: src/main/java/org/verdictdb/metastore/ScrambleMetaStore.java [hunk 1]
```diff
@@ -153,15 +153,17 @@ public void addToStore(ScrambleMetaSet scrambleMetaSet) throws VerdictDBExceptio
...
     // insert a new entry
+    String replacedString = "";
     StringBuilder insertSqls = new StringBuilder();
     for (ScrambleMeta meta : scrambleMetaSet) {
       InsertValuesQuery q = createInsertMetaQuery(meta);
       String s = QueryToSql.convert(conn.getSyntax(), q);
       LOG.debug("Adding a new scramble meta entry with the query: {}", s);
       insertSqls.append(s);
       insertSqls.append("; ");
+      replacedString = insertSqls.toString().replace(" 'timestamp  ", " timestamp '");
```
Comment (commercial-hippie) [inline src/main/java/org/verdictdb/metastore/ScrambleMetaStore.java]: There is probably a better way to do this.. I'm prepending `timestamp` to the timestamps column in the query, but becausue of where I'm doing it it gets quoted.

File: src/main/java/org/verdictdb/sqlsyntax/SqlSyntax.java [hunk 2]
```diff
@@ -95,4 +95,8 @@ public String getFallbackDefaultSchema() {
   public String getApproximateCountDistinct(String column) {
     return String.format("count (distinct %s)", column);
   }
+
+  public String getTimestampPrefix() {
```
Comment (commercial-hippie) [inline src/main/java/org/verdictdb/sqlsyntax/SqlSyntax.java]: This is related to the comment I made above..

File: src/main/java/org/verdictdb/sqlsyntax/SqlSyntaxList.java [hunk 3]
```diff
@@ -43,6 +44,10 @@ public static SqlSyntax getSyntaxFor(String dbName) {
   public static SqlSyntax getSyntaxFromConnectionString(String connectionString) {
     String dbName = connectionString.split(":")[1];
     SqlSyntax syntax = SqlSyntaxList.getSyntaxFor(dbName);
+    if (syntax == null) {
```
Comment (commercial-hippie) [inline src/main/java/org/verdictdb/sqlsyntax/SqlSyntaxList.java]: This was hack fix - I wasn't sure of the cause of the error but sometimes it wouldn't get the driver name correctly, just had to get it working!

**Comment by commercial-hippie** [comment]
@pyongjoo made some comments on the code.. Happy to update it just need a little bit of direction.

**Comment by dongyoungy** [comment]
@commercial-hippie Thank you for the comments.

I have created another pull request (#282) merging this pull request on the latest version of VerdictDB. I have addressed most of issues raised by @commercial-hippie there. Please use PR #282 for future discussion, but leave this PR open just in case. We can close both PRs once #282 gets merged into master.

**Pull request merged**